utils/python/CIME/case.py

```diff
@@ -758,7 +753,7 @@ def create_clone(self, newcase, keepexe=False, mach_dir=None, project=None):
         # user's case. However, note that, if a project is not given, the fallback will
         # be to copy it from the clone, just like other xml variables are copied.
         if project is None:
-            project = get_project()
+            project = self.get_value("PROJECT")
```

Comment:
Is this change just to make sure that the code is doing what the comment actually says it's doing?